ui/src/services/graphUtils.js

```diff
@@ -392,6 +392,6 @@ export const renderQuantileData = (
               )}</td></tr>`,
           )
           .join('')
-      : ''
+      : 'Sorry, failed to load data due to error occurs.'
```

Comment:
It shall be translated. Also I would reword it to : 
EN : `Sorry an error occurred and we failed to retrieve the nodes outpassing this threshold.`
FR : `Désolé une erreur nous empêche de retrouver les noeuds dépassant ce seuil.`

Reply:
Add the translation in 92f6b474ebcf1c6fe5a7bddb86186991c3278ce6